routes/api/stories.js

```diff
@@ -0,0 +1,69 @@
+//2
+const express = require('express');
+const router = express.Router();
+const createError = require('http-errors');
+const {asyncHandler} = require('../utils');
+const { PostLike } = require("../../db/models");
+const { restoreUser } = require('../../auth');
+
+/* Like a story */
+router.post('/stories/:id/likes', restoreUser, asyncHandler(async(req, res, next) => {
+
+        if (!res.locals.authenticated) {
+
+           /* If the user is not authenticated,
+              return a 401 error */ 
+           return next(createError(401));
+
+        } else if (res.locals.authenticated ) {
+            
+            /* If the user is authenticated,
+               create a new like */
+            const likedPost = await PostLike.create({
+                userId: res.locals.user.id,
+                postId: req.params.id
+            })
+        }
+
+        /* User has been authenticated, and
+           a like has been created.
+           Set 201 status code for response, and
+           send likedPost as json response. */
+        res.status(201).json(likedPost);
```

Comment:
`likedPost` is not in the scope of this line

Reply:
The new post like is now initialized to be passed as a json object.